### DISCUSSION TITLE
how to install in laravel 8

### DISCUSSION THREAD

**Discussion by abdelazizib** [cjmellor/approval#29] [Q&A]
I have tried several times to install the package in Laravel 8, but I couldn't do it. Is there a solution I can follow instead of executing the package installation from the beginning?

**Comment by cjmellor** [comment]
Hi,

This package was built with v9 in mind (and since its inception, has been can be used with v10).

v8 is now at EOL status and shouldn't be used.

If you really need v8 support, you can fork the package and do the work to make it work for v8, but I will never support v8 on the package.
